### DISCUSSION TITLE
Error searching messages, API responded with status 401! 	{"message":"401: Unauthorized","code":0} CoreException	{}

### DISCUSSION THREAD

**Discussion by Kingatherx** [victornpb/undiscord#522] [2. Discussions]

**Comment by 3r4y** [comment]
![image](https://github.com/victornpb/undiscord/assets/52072572/98a2e6dd-d575-4312-96de-15dd46cae0b0)
![image](https://github.com/victornpb/undiscord/assets/52072572/ede7dbfa-5336-461b-be28-148681606fdc)

Bro, due to automatic password filling, your passwords are populating in these tabs. To fill this again, press the "Fill" button (click on "Advanced Settings" if needed). Additionally, you need to do the same for the "Messages Interval." Press the "Before a Message" button and click on the last message. Undiscord will then delete all remaining messages. That's it, bro!

**Reply by m-nikita** [reply]
Thanks :)
